pyvista/core/composite.py

```diff
@@ -205,7 +205,7 @@ def bounds(self) -> BoundsLike:
             maxima = np.maximum.reduce(all_bounds)[1::2]
 
         # interleave minima and maxima for bounds
-        the_bounds = np.stack([minima, maxima]).ravel('F')
+        the_bounds = np.stack([minima, maxima]).ravel('F').tolist()
```

Comment:
I feel like it might be more efficient to explicitly form the tuple directly rather than creating a new ndarray, convert to list, then convert to tuple.  But I'm not sure.  There is another method, `item` that can convert a numpy scalar from an array to a python scalar.  I think it would come down to whether this being optimized within `tolist` outweighs the multiple container conversions.  Since `tolist` documents that it uses `item` under the hood, I have my doubts there is benefit here.

Either way, I think it is likely we will be making this decision as a whole in the other PR so it makes sense to do there.